README.md

```diff
@@ -214,7 +215,7 @@ All the data (past and coming) is available publicly in JSON:
 ### January
 
 * 20: [Voxxed Days Ticino](https://voxxeddays.com/ticino/) - Lugano (Switzerland)
-* 31-03/02: [SnowCamp](https://snowcamp.io/fr/) - Grenoble (France)
+* 31-03/02: [SnowCamp](https://snowcamp.io/) - Grenoble (France)<a href="https://snowcamp.io/"><img alt="CFP SnowCamp" src="https://img.shields.io/static/v1?label=CFP&message=until%2015-January-2024&color=green"></a>
```

Comment:
**question(non-blocking)**: they also provide a *springboard* ("tremplin") to coach potential speakers, should it be added too ?

Reply:
The "tremplin" is only a concept in France for the moment.
The link is available in the snowcamp website?

Reply:
Link for the tremplin : https://conference-hall.io/public/event/7tzYnx2FMO3J2q4N7JiE
It does not seem to be present on their website, but they shared it on their official LinkedIn : https://www.linkedin.com/feed/update/urn:li:activity:7088100326846734337/